.github/ISSUE_TEMPLATE/crd_request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Resource Support Request
+about: Help us know what resource you need is missing
+labels: services
```

Comment:
nitpick: would `resources` or `resource-requests` be clearer (same with template file title)?

Reply:
Yes, I didn't know we can rename a label but turns out you can. So I renamed `services` to `new-resource` and changed it here as well. Could you take another look?